### ISSUE TITLE
Questions about loss curve

### ISSUE THREAD

**Issue by GssUCAS** [martinarjovsky/WassersteinGAN#31]
Hi! @martinarjovsky

I have some questions about the loss curve. When I don't use the median filter operation, the loss curve is still oscillates, which describes the training is unstable, am I right??

Please help me! Thank you!

**Comment by martinarjovsky** [comment]
Not really, that's because it's a training curve (as opposed to a validation set curve)! Since the minibatches are tiny, different minibatches will have different losses, hence the oscilation. If you were to plot this values over a big validation set (say ~ 10K examples), then the oscilations would mostly dissapear. This is expensive though, so we just put it through a simple filtering :).
